### ISSUE TITLE
Implementation of `ariaActiveDesendantsElement` in `Select`and `MultiSelect` component

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3764]
Scope
---
Due to the native behavior of Web Components with its scoped ShadowDOM, it is not possible to set relationship through IDREF's between elements within different shadowDOM's. In the case of the `Select` and `MultiSelect` component, it's needed to set a relationship between the `combobox` and the selected `option` element to expose the highlighted and selected state to assistive technologies. 

This would also bring out of the box **i18n** behaviour in screen reader announcements which is currently implemented with an english only workaround.

`ariaActiveDesendantsElement` should be at least supported by latest Browsers:
- https://caniuse.com/?search=ariaActive

Notes  
---
- Also the change from sproutet ARIA option role and selected attribute to `attachInternals()` should be checked

More information about AOM and reflecting IDREF's:
- https://eisenbergeffect.medium.com/web-components-2024-winter-update-445f27e7613a
- https://github.com/WICG/webcomponents/blob/gh-pages/proposals/reference-target-explainer.md
- https://alice.pages.igalia.com/blog/how-shadow-dom-and-accessibility-are-in-conflict/#referring-into-shadow-dom

Acceptance criteria
---
- [ ] `ariaActiveDescendant` should be reflected/exposed to the accessibilityTree

Subtasks
---
- [ ] Add note to a11y docs that in Safari and Voiceover there might be issues
